src/browser/index.jsx

```diff
@@ -139,6 +137,7 @@ function handleSelectSpellCheckerLocale(locale) {
 ReactDOM.render(
   <MainPage
     teams={teams}
+    config={config}
```

Comment:
Perhaps if component child depth increases, config may be better accessed through context provider.
For now, this is simpler.